core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -93,13 +107,80 @@
       "<code>MetricReporter</code> interface allows plugging in classes that will be notified " +
       "of new metric creation. The JmxReporter is always included to register JMX statistics.";
   protected static final String METRICS_REPORTER_CLASSES_DEFAULT = "";
+  public static final String SSL_KEYSTORE_LOCATION_CONFIG = "ssl.keystore.location";
```

Comment:
I'm interested to hear @ijuma's opinion, too.

I agree that having some sort of factory to make security more convenient is a fabulous idea. And seems feasible in theory, although I haven't tried it.

The switch to Apache's `ConfigDef` will simply this a lot because we can use `SslConfigs.addClientSslSupport()` and `SaslConfigs.addClientSaslSupport()`.